src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+* Updated [Semantic Conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/specification/trace/semantic_conventions/database.md).
```

Comment:
I think we're lacking clarity here, https://github.com/open-telemetry/opentelemetry-specification/blob/v1.21.0/CHANGELOG.md#v1210-2023-05-09 clearly called out that semconv got moved to a new repo so it doesn't seem to be part of v1.21.0 release, yet we're referring to 1.21.0 database link, what does that mean and what exactly are we trying to follow?

Reply:
Let me change the question, which exact semantic convention version do we want to implement in this PR? (I don't think the goal to implement the link that you put here)

Reply:
We want to implement the breaking changes to the HTTP Semantic Conventions.
These changes are described in the v1.21.0 of the *opentelemetry-specification*.
https://github.com/open-telemetry/opentelemetry-specification/tree/v1.21.0

Reply:
FYI for all persons here.

Semantic-Conventions repo is readying their first release. The version will be v1.21.0. No ETA because they're building out the processes.
https://github.com/open-telemetry/semantic-conventions/blob/df8e53054147acf49918582bd446d0f524170071/CHANGELOG.md

Trask advised that we should link to "main" for now.

I need some time to update all the comments and links. Will need to create a new PR to correct everything that merged already.